examples/tests/youtube.test.ts

```diff
@@ -0,0 +1,22 @@
+// youtube test
```

Comment:
No comments, not useful

Reply:
Removed extra comments across multiple files. Although I kept some that I thought were useful. Lmk if you wanna delete some.